website/docs/logs.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 10
+description: Logs
+---
+
+# Logs
```

Comment:
Not sure where this can be best located.

Reply:
```suggestion
# Logging
```